### DISCUSSION TITLE
Grove - Speech Recognizer | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1607] [Q&A]
# Grove - Speech Recognizer | Seeed Studio Wiki

enter image description here

https://wiki.seeedstudio.com/Grove-Speech_Recognizer/

**Comment by GiacomoGVD** [comment]
Really complicated start the sensor. I tell hi cell many, many, many time before it takes and the other world are the same complicated. It's not a nice enough for me.
